powerlevel9k.zsh-theme

```diff
@@ -557,6 +557,15 @@ prompt_nvm() {
   $1_prompt_segment "$0" "$2" "green" "011" "${node_version:1}" 'NODE_ICON'
 }
 
+# NodeEnv Prompt
+prompt_nodeenv() {
+  local nodeenv_path="$NODE_VIRTUAL_ENV"
+  if [[ -n "$nodeenv_path" && "$NODE_VIRTUAL_ENV_DISABLE_PROMPT" != true ]]; then
```

Comment:
Is `$NODE_VIRTUAL_ENV_DISABLE_PROMPT` something common to `nodeenv` or is it specific to this Powerlevel9k prompt segment?